app/lib/additional_dependents_pdf.rb

```diff
@@ -7,8 +7,8 @@ def source_pdf_name
   end
 
   def initialize(submission, start_node: 4)
-    # For some PDF fields, use values from the database b/c the XML values are truncated or missing.
-    @xml_document = SubmissionBuilder::Ty2021::Documents::Irs1040.new(submission).document
+    # Use base because these fields do not deviate amongst xml versions
+    @xml_document = SubmissionBuilder::Ty2021::Documents::Irs1040::Base.new(submission).document
```

Comment:
ohh this is going to be interesting, because if we have sub-files that dependent on specific parent files (for example, divergenct 1040s that both fill out an 8812 but have different values), we're probably going to have to deal with that somehow within these files. 🤔